src/codechicken/chunkloader/ChunkLoaderManager.java

```diff
@@ -634,6 +634,11 @@ public static void addChunkLoader(IChickenChunkLoader loader) {
     }
 
     private static ChunkLoaderOrganiser getOrganiser(IChickenChunkLoader loader) {
+    	// Added bugfix here because IChickenChunkLoad.getWorld() is needed to get the WorldServer object for load(),
+    	// and this method has the last reference to the loader
+    	if(!loaded)
+    		ChunkLoaderManager.load(DimensionManager.getWorld(CommonUtils.getDimension(loader.getWorld())));
```

Comment:
It may be.. I'm more familiar with bukkit than i am with forge. I noticed the load method wanted a WorldServer object, and loader.getWorld() returned a normal minecraft World object. Which may sufficient with a cast.. or maybe it's an instance of it.. I wasn't sure and didn't want to take chances. I saw the dimension manager returned a WorldServer object given the dimension number, and I didn't see any way to get the dimension number from the World object that loader.getWorld() returned. I saw you used common utils earlier in the code to get the dimension number so i did it that way. There may be a more efficient way.. the important thing is the loader gets called at that point before the hash maps are used.